pkgs/top-level/all-packages.nix

```diff
@@ -17511,7 +17513,9 @@ in
 
   openldap = callPackage ../development/libraries/openldap { };
 
-  opencolorio = callPackage ../development/libraries/opencolorio { };
+  opencolorio1 = callPackage ../development/libraries/opencolorio/1.x.nix { };
+  opencolorio2 = callPackage ../development/libraries/opencolorio { };
+  opencolorio = opencolorio1;
```

Comment:
Would it make sense to make 2.x the default and pin everything that needs 1.x to `opencolorio1`?